tests/chainer_tests/functions_tests/noise_tests/test_dropout.py

```diff
@@ -10,15 +10,21 @@
 
 
 # TODO(Kenta OONO): This test fixture check types only. Add more detailed test.
+@testing.parameterize(
+    {'dtype': numpy.float16},
+    {'dtype': numpy.float32},
+    {'dtype': numpy.float64},
+)
```

Comment:
The same opinion as #1533. How about using the following function?
```python
@testing.parameterize(*testing.product({
    'dtype': [numpy.float16, numpy.float32, numpy.float64],
}))
```
Although I agree that in this case ``testing.product()`` has no meanings, it would be useful when we want to write other test functions.